images/kube-api-proxy/pkg/rewriter/discovery.go

```diff
@@ -390,35 +405,42 @@ func RestoreAggregatedGroupDiscovery(rules *RewriteRules, obj []byte) ([][]byte,
 		restoredGroupObj := []byte(fmt.Sprintf(`{"metadata":{"name":"%s", "creationTimestamp":null}}`, groupName))
 
 		// Construct an array of APIVersionDiscovery objects.
-		restoredVersions := []byte(`[]`)
+		//restoredVersions := []byte(`[]`)
```

Comment:
Old comment, looks like you can delete it

Reply:
Done